[2024-12-01T16:22:44.785+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-01T16:22:44.796+0500] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_open_weather.extract_from_api manual__2024-12-01T11:22:39.065933+00:00 [queued]>
[2024-12-01T16:22:44.802+0500] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_open_weather.extract_from_api manual__2024-12-01T11:22:39.065933+00:00 [queued]>
[2024-12-01T16:22:44.803+0500] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-01T16:22:44.893+0500] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_from_api> on 2024-12-01 11:22:39.065933+00:00
[2024-12-01T16:22:44.900+0500] {logging_mixin.py:190} WARNING - /Users/hanan-nawaz/Documents/Projects/Pakistan_AQI_Analysis/myenv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=35824) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-01T16:22:44.902+0500] {standard_task_runner.py:72} INFO - Started process 35828 to run task
[2024-12-01T16:22:44.903+0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'etl_open_weather', 'extract_from_api', 'manual__2024-12-01T11:22:39.065933+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/var/folders/qc/y2_zkt8d01zf90dfpf72dm400000gn/T/tmpa11md8u7']
[2024-12-01T16:22:44.908+0500] {standard_task_runner.py:105} INFO - Job 9: Subtask extract_from_api
[2024-12-01T16:22:44.950+0500] {task_command.py:467} INFO - Running <TaskInstance: etl_open_weather.extract_from_api manual__2024-12-01T11:22:39.065933+00:00 [running]> on host abduls-macbook-pro.local
[2024-12-01T16:22:45.020+0500] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abdul_Hanan_Nawaz' AIRFLOW_CTX_DAG_ID='etl_open_weather' AIRFLOW_CTX_TASK_ID='extract_from_api' AIRFLOW_CTX_EXECUTION_DATE='2024-12-01T11:22:39.065933+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-01T11:22:39.065933+00:00'
[2024-12-01T16:22:45.021+0500] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-01T16:22:45.053+0500] {logging_mixin.py:190} INFO - Good Morning ! Mr. Hanan
[2024-12-01T16:22:45.053+0500] {python.py:240} INFO - Done. Returned value was: None
[2024-12-01T16:22:45.059+0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-01T16:22:45.060+0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_open_weather, task_id=extract_from_api, run_id=manual__2024-12-01T11:22:39.065933+00:00, execution_date=20241201T112239, start_date=20241201T112244, end_date=20241201T112245
[2024-12-01T16:22:45.083+0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-01T16:22:45.102+0500] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-01T16:22:45.103+0500] {local_task_job_runner.py:245} INFO - ::endgroup::
